### PR TITLE
Multi same type actors

### PR DESCRIPTION
**Pull request by blackstardlb**
Added ability to set one fact to multiple actors like: 
```
{ '[ingezetene]': ['ingezetene1', 'ingezetene2'] }
```

### CODE REVIEW

**Comment by blackstardlb** [comment]
OR expressions not resolving properly

File: src/index.js [hunk 1]
```diff
@@ -339,6 +339,25 @@ class LawReg {
     }
   }
 
+  /**
+   * Extract the DIDs being referred, if the input is an OR Function or a string with IS-constructions
+   *
+   * @param {ParsedExpression | string} functionRef - Possible OR Function or string with IS-constructions
+   * @returns {string[]} - The DIDs being referred if it is an IS-construction
+   */
+  static extractDids (functionRef) {
```
Comment (pimotte) [inline src/index.js:348]: Given how the did-check is currently done in the checkFactLink function I get why you went this way, but I think it's a little funky to have the explicit check for OR in this place. Do you feel it's worth investigating if we can refactor the existing code such that we can resolve the OR using the function we already have and then delegate to existing logic to just check a single did?

**Review by pimotte — COMMENTED** [review]
One kind of fundamental remark, if your answer turns out to be "not worth it", I'm fine merging this, but I would like to consider the option.

File: src/index.js (hunk 1 above)
Comment (blackstardlb) [reply src/index.js]: I agree.

**Review by pimotte — APPROVED** [review]
:+1:

**Pull request merged**